pkg/cloud/aws/bootstrap/pod.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: aws-cloud-controller-manager
+  namespace: kube-system
+spec:
+  priorityClassName: system-cluster-critical
+  containers:
+  - args:
+    - --cloud-provider=aws
+    - --use-service-account-credentials=false
+    - --controllers=*,-service,-route # disable service and route controllers in bootstrap
+    - --kubeconfig=/etc/kubernetes/secrets/kubeconfig
+    - --leader-elect=false
+    - -v=2
+    - "2> /var/log/bootstrap-control-plane/cloud-controller-manager.log"
```

Comment:
Unfortunately the approach to store logs on bootstrap machine does not work. Neither this, nor `--log-file` option works in the CCM. I assume `--log-file` is an issue in cloud-provider. I would leave this line in, but we would need to address this issue to simplify CCM debug in bootstrap node. cc @JoelSpeed @lobziik